README.md

```diff
@@ -168,11 +168,15 @@ First off, you need to `--init` terraform on your local machine, by executing th
 
 - `../../bin/express-cli --setup-datadog`
     - Sets up datadog on the nodes and gets them ready to send metrics to Datadog Dashboard. `DD_API_KEY` env var is required for this.
-    
+
+- `../../bin/express-cli --chaos [intensity]`
+    - Adds dedicated chaos(de-peering) to the network. The `intensity` parameter is optional and can be set from `1` to `10`. If not set, `5` is used.
+
```

Comment:
Can you add instruction to stop the chaos too?

Also, would you please update the commands [here](https://www.notion.so/Testing-Toolkit-d47e098641d14c80b2e9a90b3b1b88d9) too?

Thanks

Reply:
Done ✅